src/App.jsx

```diff
@@ -1,40 +1,58 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
 export const tabs = [
-  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+  {
+    id: 'tab-1',
+    title: 'Tab 1',
+    content: 'Switch to 3 Tab',
+  },
+  {
+    id: 'tab-2',
+    title: 'Tab 2',
+    content: 'I\'m not here, look for me on 4 Tab',
+  },
+  {
+    id: 'tab-3',
+    title: 'Tab 3',
+    content: 'No, switch to 2 Tab',
+  },
+  {
+    id: 'tab-4',
+    title: 'Tab 4',
+    content: 'I caught you',
+  },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+  const [onTabSelected, setSelectedTab] = useState(tabs[0].title);
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+  function updateTabId(title) {
+    tabs.forEach((tab) => {
+      if (tab.title === title) {
+        setSelectedTabId(tab.id);
+        setSelectedTab(tab.title);
+      }
+    });
+  }
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
+  return (
+    <div className="section">
+      <h1 className="title">
+        {`Selected tab is ${onTabSelected}`}
+      </h1>
 
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-          </li>
-        </ul>
-      </div>
-
-      <div className="block" data-cy="TabContent">
-        Some text 1
-      </div>
+      <Tabs
+        tabs={tabs}
+        selectedTabId={selectedTabId}
+        selectedTab={(title) => {
+          updateTabId(title);
```

Comment:
Rename props due to task description to pass tests
![image](https://github.com/mate-academy/react_tabs-js/assets/80700247/ff4559f3-de59-4b26-bcc3-c0b0f0010bed)